_activities/pyhep.md

```diff
@@ -0,0 +1,46 @@
+---
+title: PyHEP - "Python in HEP" Series of Workshops
+author: "Eduardo Rodrigues"
+layout: default
+redirect_from:
+  - /pyhep.html
+---
+
+# PyHEP "Python in HEP" Series of Workshops
+
+The **PyHEP workshops** are a new series of workshops initiated and supported by the HSF
+with the aim to provide an environment to discuss and promote the usage of Python in the HEP community at large.
+
+
+## PyHEP 2018
+
+The first workshop, **PyHEP 2018**, will be held as a pre-CHEP event in Sofia, Bulgaria, the weekend just before the start of the [CHEP 2018 conference](http://chep2018.org/), i.e. on July 7-8 2018. It will focus on the review of where and how Python is used in our community. The workshop will also be a forum for the participants, representatives of the community, to discuss topics around the areas of work identified in the HSF [Community White Paper](/activities/cwp.html). There will be ample time for discussion.
+
+The agenda will be composed (exclusively) of plenary sessions dedicated to the following topics:
+
+1. Historical perspective and overview
+2. PyROOT and Python bindings
+3. HEP python software ecosystem
+4. Distribution and installation
+5. Analysis frameworks
+6. Python 2 versus 3
+7. Open discussion on education and training
+
+Note that there won't be any training session nor hackaton in this first workshop.
+Some level of (brief) training will come in small bites in the various presentations,
```

Comment:
Suggest "There won't be any training sessions or hackathons in this first workshop, but some level of tuition will come in...".

Reply:
OK.